criu/pie/restorer.c

```diff
@@ -296,8 +297,17 @@ static int restore_creds(struct thread_creds_args *args, int procfd)
 		return -1;
 	}
 
-	if (lsm_set_label(args->lsm_profile, procfd) < 0)
-		return -1;
+	if (lsm_type != 1) {
```

Comment:
Can we use LSMTYPE__SELINUX here? If we can't, pls write a few lines in the comment about this.

Reply:
I thought we cannot include the protobuf headers in the restorer, but did not actually verify it. I just tried it and I can use LSMTYPE__SELINUX without a problem. Will push an update.